admin-dev/themes/new-theme/js/components/entity-search-input.ts

```diff
@@ -39,19 +40,40 @@ import Bloodhound from 'typeahead.js';
  * creating a new clean form type with better templating (the tplcollection brings nearly no value as is)
  */
 export default class EntitySearchInput {
-  constructor($entitySearchInput, options) {
+  $entitySearchInput: JQuery;
```

Comment:
I wonder if we need to keep the `$` prefix now? Personally, I used it to easily identify which variables were jQuery elements vs which ones were native ones But since now they are strongly typed maybe it's not that useful anymore? 🤔

Reply:
We can keep it and it helps you to understand if the var is a jquery element anywhere in the file too, I don't mind about this